parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -356,6 +356,7 @@ MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
     sed -i "s|<searchDomainName>|{{WrapAsParameter "searchDomainName"}}|g" "/opt/azure/containers/setup-custom-search-domains.sh"
     sed -i "s|<searchDomainRealmUser>|{{WrapAsParameter "searchDomainRealmUser"}}|g" "/opt/azure/containers/setup-custom-search-domains.sh"
     sed -i "s|<searchDomainRealmPassword>|{{WrapAsParameter "searchDomainRealmPassword"}}|g" "/opt/azure/containers/setup-custom-search-domains.sh"
+    sed -i "s|<searchDomainComputerOU>|{{WrapAsParameter "searchDomainComputerOU"}}|g" "/opt/azure/containers/setup-custom-search-domains.sh"    
```

Comment:
you will need to add this line to https://github.com/Azure/acs-engine/blob/fix-multiple-agentpools/parts/k8s/kubernetesmastercustomdatavmss.yml#L384 as well

Reply:
ok thanks, will do - however my issue is that the sed substitutions do not occur during bootstrap of each machine.
see comment in #3737  for further info: 
https://github.com/Azure/acs-engine/issues/3737#issuecomment-418442112 

I'm not sure the kubelet service (which I believe is supposed to trigger this script) is working properly. Any idea why that might be?